addons/im_livechat/views/im_livechat_channel_views.xml

```diff
@@ -29,7 +29,10 @@
                         <t t-name="menu">
                             <div class="container">
                                 <a type="object" name="action_view_rating" class="dropdown-item" role="menuitem">Ratings</a>
-                                <a type="open" class="dropdown-item" role="menuitem">Configure Channel</a>
+                                <a type="open" class="dropdown-item" role="menuitem">
+                                    <span groups="im_livechat.im_livechat_group_user,!im_livechat.im_livechat_group_manager">View Channel</span>
```

Comment:
Shouldn't it be just `!im_livechat.im_livechat_group_manager`?
Condition on user is implicit through ACL for viewing the record in the first place.